chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/MacraMeasureDAO.java

```diff
@@ -68,6 +68,14 @@ public void removeAllByValue(String value) {
         entityManager.flush();
     }
 
+    public void updateSubstringInAllValues(final String oldSubstring, final String newSubstring) {
```

Comment:
The two added DAO methods are pretty specific to this task which is pretty specific to Cures and the way we are using jobs to update data at certain times. We are trying to keep these type of methods out of the codebase, so the technique is to make a dao inner class in your job that has whatever methods you need.  When Cures is done we can delete these types of jobs. Check out ModifyTestFunctionalityJob for an example.